File: tests/fixtures/gin/accounts.go
package main

import (
	"net/http"

	"github.com/gin-gonic/gin"
)

func main() {
	r := gin.Default()

	v1 := r.Group("/v1")
	{
		accounts := v1.Group("/accounts")
		{
			accounts.GET("", listAccounts)
			accounts.GET("/:accountId", getAccount)
			accounts.POST("", createAccount)
			accounts.PUT("/:accountId", updateAccount)
			accounts.DELETE("/:accountId", deleteAccount)
		}
	}

	r.Run(":8080")
}

func listAccounts(c *gin.Context) {
	c.JSON(http.StatusOK, gin.H{})
}

func getAccount(c *gin.Context) {
	accountId := c.Param("accountId")
	c.JSON(http.StatusOK, gin.H{"id": accountId})
}

func createAccount(c *gin.Context) {
	c.JSON(http.StatusCreated, gin.H{})
}

func updateAccount(c *gin.Context) {
	c.JSON(http.StatusOK, gin.H{})
}

func deleteAccount(c *gin.Context) {
	c.Status(http.StatusNoContent)
}

File: tests/fixtures/express/accounts.js
const express = require('express');
const router = express.Router();

router.get('/v1/accounts', (req, res) => {
    res.json([]);
});

router.get('/v1/accounts/:accountId', (req, res) => {
    const { accountId } = req.params;
    res.json({ id: accountId });
});

router.post('/v1/accounts', (req, res) => {
    const body = req.body;
    res.status(201).json({});
});

router.put('/v1/accounts/:accountId', (req, res) => {
    res.json({});
});

router.delete('/v1/accounts/:accountId', (req, res) => {
    res.status(204).send();
});

module.exports = router;
